pkg/types/validation/installconfig_test.go

```diff
@@ -962,16 +962,6 @@ func TestValidateInstallConfig(t *testing.T) {
 				return c
 			}(),
 		},
-		{
-			name: "unsupported manual cloud credentials mode",
-			installConfig: func() *types.InstallConfig {
-				c := validInstallConfig()
-				c.Platform = types.Platform{GCP: validGCPPlatform()}
-				c.CredentialsMode = types.ManualCredentialsMode
-				return c
-			}(),
-			expectedError: `^credentialsMode: Unsupported value: "Manual": supported values: "Mint", "Passthrough"$`,
-		},
```

Comment:
Could this test stay if changed to a platform that isn't AWS, Azure, or GCP

Reply:
It could, but it is already covered by the next test case below it.

Reply:
Ack